packages/botonic-react/src/utils.js

```diff
@@ -8,6 +11,47 @@ import { Video } from './components/video'
 import { Document } from './components/document'
 import { Location } from './components/location'
 
+export function decomposeComponent(component) {
+  let componentJSON = null
+  let carousel = []
+  try {
+    component[0].props.children.map((e, i) => {
```

Comment:
noob question:
is it possible that component has more than 1 element?

Reply:
I have tried different methods and I could not reproduce a case when a component had more than 1 element was with the Location, but i just refactor the Location component because we are only using a URL button and the component had trash code. Now, all the components should only have 1 element

Reply:
cool, thanks